### PR TITLE
Phase2-hgx294A Add layeroffset attribute to 6-inch wafer geometry files for HGCal

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add layeroffset attribute to 6-inch wafer geometry files for HGCal

#### PR validation:

Tested using the workflows for HGCal TB which uses 6-inch wafer

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by srimanob** [comment]
please abort

To pass, you will need https://github.com/cms-sw/cmssw/pull/36021 for now.

**Comment by srimanob** [comment]
@cmsbuild please test with https://github.com/cms-sw/cmssw/pull/36021

**Comment by bsunanda** [comment]
@cmsbuild please test with #36021

File: Geometry/HGCalCommonData/src/HGCalGeomParameters.cc [hunk 1]
```diff
@@ -1223,6 +1228,14 @@ void HGCalGeomParameters::loadSpecParsHexagon(const cms::DDFilteredView& fv,
   php.cellSize_ = fv.get<std::vector<double> >(sdTag3, "CellSize");
   rescale(php.cellSize_, HGCalParameters::k_ScaleFromDD4hepToG4);
 
+  // Layer Offset
+  const auto& dummy2 = fv.get<std::vector<double> >(sdTag1, "LayerOffset");
```
Comment (cvuosalo) [inline Geometry/HGCalCommonData/src/HGCalGeomParameters.cc:1232]: `layerOffset` would be a more instructive name for this constant than `dummy2`.

**Comment by cvuosalo** [comment]
+1

**Comment by srimanob** [comment]
+Upgrade

**Comment by qliphy** [comment]
+1

**Pull request merged**